src/gtp/driver.rs

```diff
@@ -72,6 +62,7 @@ impl Driver {
                 Command::TimeLeft           => print!("= \n\n"),
                 Command::TimeSettings       => print!("= \n\n"),
                 Command::Version            => print!("= {}\n\n", engine_version),
+                Command::ErrorMessage(e)    => print!("? {}\n\n", e),
```

Comment:
Would it be possible to get rid of the `_` match case? Then the compiler could check here too if the match is exhaustive.